### PR TITLE
Completed Typescript Conditionals Exercise-Allyson Greven

### PR DESCRIPTION
**Pull request by allygreven**
Screenshot 2024-10-24 at 4 17 34 PM (2)

### CODE REVIEW

File: typescript-conditionals/main.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            recommendMovie
+ */
+function isUnderFive(number: number): boolean {
+  return number < 5;
+}
+console.log('isUnderFive: ', isUnderFive(3));
+/// /////////////////////////////////////////
+function isEven(number: number): boolean {
+  return number >= 2 + 2;
+}
```
Comment (shawnkost) [inline typescript-conditionals/main.ts:18]: This isn't accounting for even numbers, this is just checking if `number` is greater than or equal to 4. You should be utilizing the `%` operator, to check if a number is even or odd

File: typescript-conditionals/main.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+console.log(
+  'isisOldEnoughToDrinkAndDrive: ',
+  isOldEnoughToDrinkAndDrive(person)
+);
+/// //////////////////////////////////////////////////
+
+function categorizeAcidity(pH: number): string {
+  if (pH < 7) {
+    return 'acid';
```
Comment (shawnkost) [inline typescript-conditionals/main.ts:71]: Are all numbers less than 0 considered acid? What about negative numbers?

File: typescript-conditionals/main.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 7 && pH < 14) {
+    return 'base';
```
Comment (shawnkost) [inline typescript-conditionals/main.ts:75]: What about 14? Is 14 not considered a base?

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes here Ally!

**Comment by allygreven** [comment]
Screenshot 2024-10-25 at 12 18 13 PM (2)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**